app/controllers/friendships_controller.rb

```diff
@@ -5,6 +5,7 @@ class FriendshipsController < ApplicationController
   before_action :set_user
```

Comment:
Fixed the user sign up here: https://github.com/danfstucky/LeagueSpec/pull/47/commits/7f23c57ea9676451acaf6081ea991042fc89f9a0
For this error, I think I will add the lol_connections file back as a temporary fix, which will resolve it.  Read my comment above for my reasoning on this.  I will post a commit with that change later.